locales/en/common.json

```diff
@@ -0,0 +1,4 @@
+{
+    "landingPageHeroHeading_1": "Building the future of",
+    "landingPageHeroHeading_2": "Event-Driven Architectures (EDA)"
+}
```

Comment:
I rather prefer to name it as `landing-page` namespace, not `common` - common can be used for navigation, footer or something like that.

Also, if we will have a `landing-page` namespace then we don't need the `landingPage` prefix for translation labels. We can go with `main`, `subMain` etc sections, something like:

```json
{
  "main": {
    "header": "Building the future of"
    "subHeader": "Event-Driven Architectures (EDA)"
  },
  ...
}
```

I think that you understand that "flow" :) So if you need long labels, you can split them to the nested JSON objects :)

Reply:
Got it :)